docs/topics/wasm/wasm-publish-artifacts.md

```diff
@@ -0,0 +1,57 @@
+[//]: # (title: Publish artifacts)
+
+Kotlin/Wasm can generate artifacts to publish as a site on static hosting services. This tutorial
+demonstrates how to generate artifacts for a [Compose Multiplatform](https://www.jetbrains.com/lp/compose-multiplatform/)
+ app and publish them as a site on [GitHub pages](https://pages.github.com/).
+
+## Before you start
+
+1. Download and install the latest version of [IntelliJ IDEA](https://www.jetbrains.com/idea/).
+2. Clone the [Kotlin/Wasm examples](https://github.com/Kotlin/kotlin-wasm-examples/tree/main) repository
+   by selecting **File** | **New** | **Project from Version Control** in IntelliJ IDEA.
+
+   You can also clone it from the command line:
+
+   ```bash
+   git clone git@github.com:Kotlin/kotlin-wasm-examples.git
+   ```
+
+## Generate artifacts
+
+1. Open the **Gradle** tool window: **View** | **Tool Windows** | **Gradle**.
+2. In the **compose-example** | **Tasks** | **kotlin browser**, select and run the **wasmJsBrowserDistribution** task.
+
+   ![Run the Gradle task](wasm-gradle-task-window-compose.png){width=650}
+
+   Alternatively, you can run the following command in Terminal from the project directory:
```

Comment:
If this is intended for people who are new to Gradle, it would be worth mentioning being in a `compose-example` directory as opposed to "project directory"